packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -638,7 +629,30 @@ export default class Meetings extends WebexPlugin {
                 if (createdMeeting && createdMeeting.on) {
                   // Create a destruction event for the meeting.
                   createdMeeting.on(EVENTS.DESTROY_MEETING, (payload) => {
-                    this.destroy(createdMeeting, payload.reason);
+                    if (this.config.autoUploadLogs) {
+                      this.uploadLogs({
+                        callStart: meeting.locusInfo?.fullState?.lastActive || undefined,
```

Comment:
You don't need to add the or statement, the optional chaining handles it for us.

```suggestion
                        callStart: meeting.locusInfo?.fullState?.lastActive,
```